addon/components/o-s-s/attribute-text.ts

```diff
@@ -0,0 +1,32 @@
+import { assert } from '@ember/debug';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface OSSAttributeTextArgs {
+  label: string;
+  value?: string;
+  copyable?: boolean;
+  tooltip?: string;
+}
+
+export default class OSSAttributeText extends Component<OSSAttributeTextArgs> {
+  @tracked hasError: boolean = false;
+  @tracked displayInitials: boolean = false;
+
+  constructor(owner: unknown, args: OSSAttributeTextArgs) {
+    super(owner, args);
+  }
+
+  get label(): string {
+    assert(`[component][OSS::Attribute::Text] label is required `, this.args.label);
```

Comment:
Set the `assert` in the constructor. You can remove this getter and use `@label` in the template directly